docker-compose/Makefile

```diff
@@ -340,7 +342,8 @@ init-couchdb:
 .PHONY: init-whisk-cli
 init-whisk-cli:
 	echo "waiting for the Whisk controller to come up ... "
-	until $$(curl --output /dev/null --silent --head --fail http://$(DOCKER_HOST_IP):8888/ping); do printf '.'; sleep 5; done
+	# removed --head from this curl because of an upstream change https://github.com/apache/incubator-openwhisk/pull/4494
+	until $$(curl --output /dev/null --silent --fail http://$(DOCKER_HOST_IP):8888/ping); do printf '.'; sleep 5; done
```

Comment:
@SugandhaAgrawal  the `head` related change in this PR https://github.com/apache/incubator-openwhisk/pull/4494/files breaks all head requests to the controller. We will need an explicit `head` handler which we should restore.

Reply:
opened https://github.com/apache/incubator-openwhisk/issues/4550.